Sources/Core/AWSClientRuntime/Regions/DefaultRegionResolver.swift

```diff
@@ -13,7 +13,6 @@ public struct DefaultRegionResolver: RegionResolver {
 
     public init(fileBasedConfigurationProvider: @escaping FileBasedConfigurationProviding) throws {
         self.providers = [
-            BundleRegionProvider(),
             EnvironmentRegionProvider(),
             ProfileRegionProvider(fileBasedConfigurationProvider: fileBasedConfigurationProvider),
             try IMDSRegionProvider()
```

Comment:
After removing `BundleRegionProvider`, are the rest of these region providers complete & in the right order?

Reply:
Yes. 
`nY63AaBMJcoU/Cross-SDK-Feature-Matrix-Items-Deprecated-or-Without-SEPs#temp:C:UcSa0a343b6b2034712914a2965f`